runelite-client/src/test/java/net/runelite/client/plugins/screenshot/ScreenshotPluginTest.java

```diff
@@ -231,4 +233,13 @@ public void testHunterLevel2()
 
 		verify(drawManager).requestNextFrameListener(Matchers.any(Consumer.class));
 	}
+
+	@Test
+	public void testPvPKill()
+	{
+		ChatMessage chatMessageEvent = new ChatMessage(SERVER, "", PVP_KILL, null);
+		screenshotPlugin.onChatMessage(chatMessageEvent);
+
+		assertEquals("Nerdpuff", screenshotPlugin.getPvPKillName());
```

Comment:
This should also verify that the screenshot plugin attempts to get a screenshot, like the other tests in this file.

Reply:
Not too sure I understand how to. 

The `PvPKillName` is set in the same block that calls `takeScreenshot(fileName)`.

Doesn't this make the test 'semi' rigid? We're testing that the name is pulled correctly?

I could test the full file name but that would involve mocking Date?